testsupport/user_assertions.go

```diff
@@ -219,11 +223,18 @@ func VerifySpaceRelatedResources(t *testing.T, awaitilities wait.Awaitilities, u
 	tiers.VerifyNSTemplateSet(t, hostAwait, memberAwait, nsTemplateSet, tierChecks)
 }
 
-func ExpectedUserAccount(userID string, originalSub string) toolchainv1alpha1.UserAccountSpec {
+func ExpectedUserAccount(sub, userID, accountID, originalSub, email string) toolchainv1alpha1.UserAccountSpec {
```

Comment:
I wonder if this function could use PropogatedClaims param instead?

```suggestion
func ExpectedUserAccount(claims *PropogatedClaims) toolchainv1alpha1.UserAccountSpec {
```

WDYT?

Reply:
Good idea, implemented.